templates/repo/clone_buttons.tmpl

```diff
@@ -18,9 +18,18 @@
 		{{svg "octicon-paste"}}
 	</button>
 {{end}}
+{{if $.DisableHTTP}}
+	<script defer>
+		const httpsDisabled = true;
+	</script>
+{{else}}
+	<script defer>
+		const httpsDisabled = false;
+	</script>
+{{end}}
 {{if not (and $.DisableHTTP $.DisableSSH)}}
 	<script defer>
-		const isSSH = localStorage.getItem('repo-clone-protocol') === 'ssh';
+		const isSSH = httpsDisabled || localStorage.getItem('repo-clone-protocol') === 'ssh';
```

Comment:
```suggestion
		const isSSH = {{if $.DisableHTTP}}true || {{end}}localStorage.getItem('repo-clone-protocol') === 'ssh';
```
A suggestion: this changes this PR to one line vs 10. Feel free to take it or discard, although if discarded then the above code chunk could have some de-duplication.

Reply:
that will fail our lint

Reply:
(I tried this orignally)

Reply:
But how about when they disabled both HTTP and SSH? Just use the UI?

Reply:
😭 ah, I see.

Reply:
We can bypass the JS linter for some code blocks. See the `head.tmpl`

![image](https://user-images.githubusercontent.com/2114189/157183189-7b4e80f1-9a33-45bd-8b02-10bd9e32f89f.png)